astroquery/query.py

```diff
@@ -59,11 +59,12 @@ def timeout(self, value):
             self._timeout = value
 
     def request(self, session, cache_location=None, stream=False,
-                auth=None, verify=True):
+                auth=None, verify=True, allow_redirects=True):
```

Comment:
Changes to this file messes up the tests for the rest of the modules. Could you double check?

Reply:
OK weird, I don't see why that would...